packages/react/src/auto/hooks/useRelatedModelManager.tsx

```diff
@@ -0,0 +1,39 @@
+import { useApi } from "../../GadgetProvider.js";
+import { GadgetHasManyConfig } from "../../internal/gql/graphql.js";
+import { getModelManager } from "../../utils.js";
+import { useFieldMetadata } from "./useFieldMetadata.js";
+
+export const useRelatedModelManager = (props: {
+  field: string; // Field API identifier
```

Comment:
I think this API is gonna get dicey if we ever do deep relationship nesting, where you can adjust a model two relationships over because so much of this thing's behavior is determined by the context. What do you think about accepting the metadata as an argument to make it a bit clearer and adjustable?

Reply:
Updated this to take in the field config as a prop instead of a field-apiID so that being within an AutoForm is not required. This calls a super general hook that takes in `model: {apiId, namespace}`